test/test_runlib.py

```diff
@@ -216,6 +217,75 @@ def test_exclude_subsubdir(self):
     self.assertListEqual(sorted(artifacts_dict.keys()),
         sorted(["foo", "bar", "subdir/foosub1", "subdir/foosub2"]))
 
+class TestInTotoRecordRun(unittest.TestCase):
+  """"Test in_toto_run(name, material_list, product_list,
+  link_cmd_args, key=False, record_byproducts=False). """
```

Comment:
Hm. It is the TestCase level description. All test functions of this TestCase test the in_toto_run function. How it is tested is explained in each test function's docstring. But I can elaborate if you want me to.